### DISCUSSION TITLE
152

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#91] [General]
# 潮流周刊第152期 - 赛博红桥

封面图来源于在在湖广会馆拍的远处的长桥，黄色的建设和红色的桥看着还挺有感觉的，我很喜欢这个配色。

https://weekly.tw93.fun/posts/152-%E8%B5%9B%E5%8D%9A%E7%BA%A2%E6%A1%A5

**Comment by fzdwx** [comment]
`How to be a Programmer 中文版本` 这个链接错了

**Reply by JasonSubMara** [reply]
https://braydie.gitbooks.io/how-to-be-a-programmer/content/zh/

**Reply by tw93** [reply]
已经更新 谢谢指出
